SimCalorimetry/EcalSimAlgos/src/ComponentShape.cc

```diff
@@ -0,0 +1,57 @@
+#include <cmath>
+
+#include "SimCalorimetry/EcalSimAlgos/interface/ComponentShape.h"
+
+// #define component_shape_debug 1
+
+void ComponentShape::fillShape(float& time_interval,
+                               double& m_thresh,
+                               EcalShapeBase::DVec& aVec,
+                               const edm::EventSetup* es) const {
+#ifdef component_shape_debug
```

Comment:
all these `cout`s within `#ifdef component_shape_debug` could be potentially replaced by edm::LogDebug and the `#ifdef` would not be needed anymore (and same for all the lines below)